common/views/components/ArchiveTree/ArchiveTree.js

```diff
@@ -715,6 +739,15 @@ const NestedList = ({
   );
 };
 
+const ButtonWrap = styled(Space).attrs({
+  v: { size: 'm', properties: ['padding-top', 'padding-bottom'] },
+})`
+  button {
+    width: 100%;
+    justify-content: center;
```

Comment:
Does `ButtonSolid` not center its own content?

Reply:
doesn't appear to no